src/BlockSettings.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace P4\MasterTheme;
+
+use P4\MasterTheme\Migrations\M001EnableEnFormFeature;
+use P4\MasterTheme\Settings\Features;
+
+/**
+ * Class BlockSettings.
+ *
+ * This class is used to handle blocks configuration.
+ */
+class BlockSettings
+{
+    private const BETA_BLOCKS_FEATURE = 'beta_blocks';
+
+    private const ALL_BLOCKS_FEATURE = 'allow_all_blocks';
+
+    private const CORE_BLOCKS_PREFIX = "core";
+
+    private const CORE_EMBED_BLOCKS_PREFIX = "core-embed";
+
+    private const P4_BLOCKS_PREFIX = "planet4-blocks";
+
+    private const P4_TEMPLATES_PREFIX = "planet4-block-templates";
+
+    private const HUBSPOT_FORMS_BLOCK = 'leadin/hubspot-form-block';
+
+    private const GRAVITY_FORMS_BLOCK = 'gravityforms/form';
+
+    private const POST_BLOCK_TYPES = [
+        self::P4_BLOCKS_PREFIX . '/accordion',
+        self::P4_BLOCKS_PREFIX . '/articles',
+        self::P4_BLOCKS_PREFIX . '/counter',
+        self::P4_BLOCKS_PREFIX . '/gallery',
+        self::P4_BLOCKS_PREFIX . '/social-media',
+        self::P4_BLOCKS_PREFIX . '/spreadsheet',
+        self::P4_BLOCKS_PREFIX . '/take-action-boxout',
+        self::P4_BLOCKS_PREFIX . '/timeline',
+        self::HUBSPOT_FORMS_BLOCK,
+        self::GRAVITY_FORMS_BLOCK,
+    ];
+
+    // pages allow all block types.
```

Comment:
To answer your question, I think we can just remove these comments about allowing all blocks.

Reply:
Done!